pkg/helm/helm.go

```diff
@@ -105,10 +105,16 @@ func (h Client) Cache() (string, error) {
 // UpdateDependencies will update dependencies for a given release if it is stored locally (i.e. pulled from git)
 func (h Client) UpdateDependencies(path string) error {
 	klog.V(5).Infof("updating chart dependencies for %s", path)
-	_, stdErr, _ := h.Exec("dependency", "update", path)
-	if stdErr != "" {
-		return fmt.Errorf("error running helm dependency update: %s", stdErr)
+	_, stdErrUpdate, _ := h.Exec("dependency", "update", path)
+	if stdErrUpdate != "" {
+		return fmt.Errorf("error running helm dependency update: %s", stdErrUpdate)
+	}
+
+	_, stdErrBuild, _ := h.Exec("dependency", "build", path)
```

Comment:
I'm not convinced we need this, but I was trying to fix something. @lucasreed do you know anything about this command?

Reply:
specifically the `dependency build` command? I'm not sure, I thought `helm dependency update` would do what we needed 🤷